content/blog/validation-split-as-3-way-split/index.Rmd

```diff
@@ -0,0 +1,151 @@
+---
+output: hugodown::hugo_document
+
+slug: validation-split-as-3-way-split
+title: New interface to validation splits
```

Comment:
We've talked about it as a new interface but I'm not sure this makes for the best title. Should this be something more general / less jargon-y like "A new way to make a validation split"? (or set?)